desc/compute/utils.py

```diff
@@ -140,11 +164,10 @@ def _compute(
         data = data_index[parameterization][name]["fun"](
             params=params, transforms=transforms, profiles=profiles, data=data, **kwargs
         )
-
     return data
 
 
-def get_data_deps(keys, obj, has_axis=False):
+def get_data_deps(keys, obj, has_axis=False, basis="rpz"):
```

Comment:
For a future PR: The variable we call `keys` here and in the other util functions below is the same as what we call `names` in the `compute` functions above and in other files. We should probably rename for consistency across the code.